traffic_portal/app/src/common/modules/form/_form.scss

```diff
@@ -222,11 +222,15 @@ div.helptext aside.warning {
 		}
 	}
 
-	p {
+	p, dl, dt {
```

Comment:
since the only permitted parent of a `<dt>` is a `<dl>` (or a `<div>` that is a direct child of a `<dl>` in HTML 5.2), it shouldn't need explicit background setting, since the initial value for `background` is `none 0% 0% auto auto repeat padding-box border-box scroll transparent` (`transparent` being the important part there) for all display elements.

This isn't a big deal, though, since the margins and padding look fine, so you wanna keep that around. I honestly wouldn't bother changing it at this point, just wanted to point it out for next time.

Reply:
In the future, should I create a separate scope for just the `<dt>` to get the desired padding/marging? Perhaps:

```css
p, dl {
  background: #ffedcc;
  padding: 5px;
  margin-bottom: 0;
}

dt {
  padding: 5px;
  margin-bottom: 0;
}
```

Reply:
I would separate out the background so that it's smaller and doesn't repeat itself:
```scss
p, dl {
  background: #ffedcc;
}
p, dl, dt {
  padding: 5px;
  margin-bottom: 0;
}
```